File: JavaScript/aula6/exe5.js
// Solicitar um nome e o numero de vezes que deseja exibi-lo.
let nome = prompt('Diga nome')
let vezes=Number(prompt('Numero de vezes'));
let i=0;
while(i <=vezes){
    document.write('<br>',nome)
    i++
}

File: Python/aula3/exe5.py
#Solicite um nome ao usuário e peça a quantidade de vezes que ele deseja exibi-lo na tela. Utilize um loop while.
nome = input('Um nome: ')
exibe= int(input('Quer exibir quantas vezes?'))
i=0
while i < exibe:
    print(nome)
    i+=1